installing/installing_openstack/installing-openstack-user-sr-iov.adoc

```diff
@@ -82,13 +82,12 @@ include::modules/cluster-telemetry.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
 == Additional resources
```

Comment:
Use unique ID for for "Additional resources" titles in assemblies. Refer to the guidance on [Anchoring "Prerequisites", "Additional resources", and "Next steps" titles in assemblies](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies)